packages/conventional-commit-lint/src/utils.ts

```diff
@@ -0,0 +1,213 @@
+    // author edited the PR title in an intention to please this bot.
+    // Let them know how to use the PR title for the commit history.
+    if (
+      usingCommitMessage &&
+      commits[0].commit.message.split('\n')[0] !== pull_request.title
+    ) {
+      logger.info(
+        `commit message: ${commits[0].commit.message.split('\n')[0]}`
+      );
+      logger.info(`PR title: ${pull_request.title}`);
+      const owner = pull_request.base.repo.owner?.login;
+      const repo = context.payload.repository.name;
+      const prNumber = pull_request.number;
+      const installationId = context.payload.installation?.id;
+      const message =
+        '🤖 I detect that the PR title and the commit message' +
+        " differ and there's only one commit. To use the PR title for the" +
+        " commit history, you can use Github's automerge feature with" +
+        ' squashing, or use `automerge` label. Good luck human!\n\n' +
+        ' -- conventional-commit-lint bot\n' +
+        'https://conventionalcommits.org/';
+      try {
+        await addOrUpdateIssueComment(
+          context.octokit,
+          owner as string,
+          repo,
+          prNumber,
+          installationId as number,
+          message,
+          false
+        );
+      } catch (err) {
+        // This is a solely convenience feature, so we ignore errors.
+        logger.warn(`Failed to add a comment: ${err}`);
+      }
+    }
+  }
+
+  // post the status of commit linting to the PR, using:
+  // https://developer.github.com/v3/checks/
+  let checkParams = context.repo({
+    name: 'conventionalcommits.org',
+    conclusion: 'success' as Conclusion,
+    // commit.sha can be null as per GitHub's OpenAPI spec  ¯\_(ツ)_/¯
+    head_sha: commits[commits.length - 1].sha as string,
+  });
+
+  if (lintError) {
+    let summary =
+      'Some of your commit messages failed linting.\n\nVisit [conventionalcommits.org](https://conventionalcommits.org) to learn our conventions.\n\n';
+    if (commits.length === 1) {
+      summary +=
+        'Run `git commit --amend` and edit your message to match Conventional Commit guidelines.';
+    } else {
+      summary +=
+        'edit your pull request title to match Conventional Commit guidelines.';
+    }
+
+    checkParams = context.repo({
+      head_sha: commits[commits.length - 1].sha as string,
+      conclusion: 'failure' as Conclusion,
+      name: 'conventionalcommits.org',
+      output: {
+        title: 'Commit message did not follow Conventional Commits',
+        summary,
+        text,
+      },
+    });
+  }
+
+  await context.octokit.checks.create(checkParams);
+}
+
+// export function lint_wrapper(message: string, rules: any): ILint {
+//   // support "foo!: message" syntax, see:
+//   // https://github.com/conventional-changelog/commitlint/pull/767
+//   message = message.replace('!:', ':');
+
+//   return lint(message.toLowerCase(), rules);
+// }
```

Comment:
Let's delete this logic if it's not used.

Reply:
Done